src/routes/accounts.js

```diff
@@ -0,0 +1,114 @@
+import { list, readOne, deleteOne, deleteMany, patchOne, createOne } from 'mongoose-crudl'
+
+import allowAccessTo from 'bearer-jwt-auth'
+import AccountModel from '../models/Account.js'
+import UserModel from '../models/User.js'
+import Email from '../helpers/Email'
+import { ConflictError } from 'standard-api-errors'
+import fs from 'fs'
+import crypto from 'crypto'
+import handlebars from 'handlebars'
+import path from 'path'
+import jwt from 'jsonwebtoken'
+import { fileURLToPath } from 'url'
+const __dirname = path.dirname(fileURLToPath(import.meta.url))
+const registration = fs.readFileSync(path.join(__dirname, '..', 'email-templates', 'registration.html'), 'utf8')
+
+export default (apiServer) => {
+  const secrets = process.env.SECRETS.split(' ')
+
+  apiServer.get('/v1/accounts/check-availability', async req => {
+    let availability = false
+    const response = await list(AccountModel, { urlFriendlyName: req.query.urlFriendlyName })
+    if (response.result.count > 0) {
+      availability = true
+    }
+    return {
+      status: availability ? 200 : 404,
```

Comment:
I think we should send 200 in both cases, since it is not really an error if a name is not available, because that's exactly that we want to check.

But when someone tries to create an account with an url friendly name that is taken, we definitely have to send back a conflict error.